solutions/python_webserver/Makefile

```diff
@@ -20,10 +20,16 @@ appdir:
 run:
 	test -f server.pid && kill -9 `cat server.pid` || true
 	$(MYST_EXEC) $(OPTS) rootfs /usr/local/bin/python3 /app/hello_server.py & echo $$! > server.pid
-	sleep 20
+	sleep 35
```

Comment:
I'd rather pass in `--max-affinity-cpus`, if that's the cause of the slow down, to save every pipeline a few seconds.

Reply:
No, this is due to my bare metal machine being slower.

Reply:
I updated the ``python_webserver`` and ``aspnet`` solutions to wait on an output string from the server rather than sleeping for a fixed period.